src/tlo/methods/healthsystem.py

```diff
@@ -2976,11 +3157,75 @@ def apply(self, population):
             matches = re.match(pattern, officer)
             # Extract ID and officer type from
             facility_id = int(matches.group(1))
-            district = self.module._facility_by_facility_id[facility_id].district
+            district = self.module._facility_by_facility_id[facility_id].district  # todo: check if district callable
```

Comment:
Hi Margheria @marghe-molaro, just raise a small todo task here: to check if _facility_by_facility_id[facility_id] has the attribute district and this line can run successfully.

If not, then a small fix might be:  district = self.module._facility_by_facility_id[facility_id].name.split('_')[-1], as the facility name info has district indicated.